distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1201,7 +1202,7 @@ type Table
            problem is reported.
     @group_by Widget_Helpers.make_column_name_vector_selector
     @order_by Widget_Helpers.make_order_by_selector
-    add_row_number : Text -> Integer -> Integer -> Vector (Text | Integer | Column_Selector) | Text | Integer | Column_Selector -> Vector (Text | Sort_Column) | Text | Sort_Column -> Problem_Behavior -> Table
```

Comment:
Why are these removed? They are valid inputs IIRC and the widget should be based on the annotation anyway.

Is it due to a bug in dropdowns logic?

Reply:
it should be but isn't yet - should be fixed one day but need to work around for now.

Reply:
Do we have a ticket for it so that we can keep track of it?

IMO it may be worth to add a TODO note near the signature - as I assume once the bug is fixed, we should fixe the signature to allow this case.

Reply:
No need for a todo as I don't see us needing to revert this change. 

I'll raise the drop-down issues as a ticket.